pytorch_pfn_extras/training/extension.py

```diff
@@ -164,7 +159,7 @@ def make_extension(
         initializer=lambda manager: None,
         on_error=lambda manager, exc, tb: None,
 ):
-    """Decorator to make given functions into extensions.
+    """Decorator to make given functions into Extension object.
```

Comment:
The decorator no longer returns Extension object.
```suggestion
    """Decorator to make given function into an extension.
```